byzanz-gui

```diff
@@ -3,6 +3,7 @@
 # AUTHOR:   (c) Rob W 2012
 #           modified by MHC (http://askubuntu.com/users/81372/mhc)
 #           and by Rodolfo Carvalho
+#			and by Edridge D'Souza
```

Comment:
I think here we might have a case of tabs x spaces, that's why it looks misaligned?